specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/blob.json

```diff
@@ -1170,6 +1174,15 @@
         }
       }
     },
+    "ChangeFeed": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
```

Comment:
booleans are less preferred as it becomes a breaking change if you need to change to an enum later. If you model this as something like '"state":[enabled|disabled]' you can add tertiary states to the enum later, without a breaking change. Something to consider.